scripts/start.sh

```diff
@@ -254,7 +259,9 @@ get_credentials() {
 
   http_2xx_or_4xx --ignore-stdin --check-status POST "${reposerver}/api/v1/user_repo" "${namespace}"
 
-  id=$(http --ignore-stdin --check-status --print=h GET http://tuf-reposerver.ota.local/api/v1/user_repo/root.json x-ats-namespace:default | grep x-ats-tuf-repo-id | awk '{print $2}' | tr -d '\r')
+  sleep 5s
```

Comment:
I still get the 423, but retrying works fine for me. I'd prefer a better way to wait for the resource to become available (we do something like that in the aktualizr tests) but for now I don't mind this.